app/services/users/delete_activity.rb

```diff
@@ -12,6 +12,34 @@ def call(user)
       user.mentions.delete_all
       user.badge_achievements.delete_all
       user.github_repos.delete_all
+
+      user.access_grants.delete_all
+      user.access_tokens.delete_all
+      user.created_podcasts.update_all(creator_id: nil)
+
+      user.blocker_blocks.delete_all
+      user.blocked_blocks. delete_all
+      user.webhook_endpoints.delete_all
+      user.authored_notes.delete_all
+      user.notes.delete_all
+      user.backup_data.delete_all
+      user.collections.delete_all
+      user.credits.delete_all
+      user.display_ad_events.delete_all
+      user.email_messages.delete_all
+      user.html_variants.delete_all
+      user.organization_memberships.delete_all
+      user.page_views.delete_all
+      user.poll_skips.delete_all
+      user.poll_votes.delete_all
+      user.pro_membership.delete if user.pro_membership.present?
+      user.profile_pins.delete_all
+      user.rating_votes.delete_all
+      user.tweets.delete_all
+
+      user.offender_feedback_messages.delete_all
```

Comment:
Not sure how `feedback_messages` are used currently. Please, tell me if we need to keep them.

Reply:
These are essentially the abuse reports that someone makes. I think it would be safer to remove the lines, but it's not a big deal since I don't think any `feedback_messages` are actually associated to any user currently.